frontend/src/components/widgets/TextArea/TextArea.tsx

```diff
@@ -66,10 +66,17 @@ class TextArea extends React.PureComponent<Props, State> {
   }
 
   private onChange = (e: React.ChangeEvent<HTMLTextAreaElement>): void => {
-    this.setState({
-      dirty: true,
-      value: e.target.value,
-    })
+    const { value } = e.target
+    const { element } = this.props
+
+    const maxChars = element.get("maxChars")
+
+    if (!maxChars || value.length <= maxChars) {
+      this.setState({
+        dirty: true,
+        value,
+      })
+    }
```

Comment:
We should show an indication that there are too many characters, otherwise the user will have no idea what's going on.

The simplest solution would be for the "press enter" text to change to "Too many characters" and become red.

Or, even better, we could add a counter like this: https://codepen.io/zabielski/pen/gPPywv

Reply:
Resolved with a counter